src/pudl/transform/eia860.py

```diff
@@ -47,10 +46,10 @@ def ownership(eia860_dfs, eia860_transformed_dfs):
     )
 
     if (min(own_df.report_date.dt.year)
-            < min(pc.WORKING_PARTITIONS['eia860']['years'])):
+            < min(Eia860Settings.working_partitions['years'])):
```

Comment:
For checks like these where we're comparing against a global value that pertains to a DataSource (here the `eia860`) it seems like it makes sense to refer directly to the DataSource, not to a copy of its information that's stored in the Settings object. The `DataSource` is only referenced by the Settings for the purpose of validating whatever data partitions etc. are passed in to generate the Settings, while the `DataSource` is the original source of truth about what the full range of allowable years/states/etc. is.